dramatiq/broker.py

```diff
@@ -23,7 +25,7 @@
 global_broker = None
 
 
-def get_broker() -> "Broker":
```

Comment:
I believe this cannot return `None`

Reply:
@CaselIT I think I found a better way out.. if we cast the type of global_broker to "Broker" it now stop type-checking errors and enables proper evaluation of the global variable.

Reply:
Fixed also.